packages/xchain-litecoin/src/node-api.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios'
+import { TxBroadcastResponse } from './types/node-api-types'
+import { BroadcastTxParams } from './types/common'
+
+/**
+ * Broadcast transaction.
+ *
+ * @see https://developer.bitcoin.org/reference/rpc/sendrawtransaction.html
+ *
+ * @returns {string} Transaction ID.
+ */
+export const broadcastTx = async ({ txHex, auth, nodeUrl }: BroadcastTxParams): Promise<string> => {
+  try {
+    const uniqueId = new Date().getTime().toString() // for unique id
+    const response: TxBroadcastResponse = (
+      await axios.post(
+        nodeUrl,
+        {
+          jsonrpc: '2.0',
+          method: 'sendrawtransaction',
+          params: [txHex],
+          id: uniqueId,
+        },
+        {
+          auth: {
+            username: auth.userName,
+            password: auth.password,
+          },
```

Comment:
Don't add `auth` if an user does not provide i (see previous comment)

[nitpick] How about to change `NodeAuth` to `{username: string, password: string}` to have just `auth` here ^.